### ISSUE TITLE
IndexError occurred when processing 'ええぞ〜〜〜'

### ISSUE THREAD

**Issue by Hiroshiba** [megagonlabs/ginza#63]
I tried to process 'ええぞ〜〜〜' text, ginza command raised error such as: `IndexError: [E040] Attempt to access token at 2, max length 2.`

<details><summary>(more message)</summary>

```bash
mode is C
disabling sentence separator
Traceback (most recent call last):
  File "/Users/me/.pyenv/versions/write_novel/bin/ginza", line 10, in <module>
    sys.exit(main())
  File "/path/to//python3.7/site-packages/ginza/command_line.py", line 195, in main
    plac.call(run)
  File "/path/to//python3.7/site-packages/plac_core.py", line 328, in call
    cmd, result = parser.consume(arglist)
  File "/path/to//python3.7/site-packages/plac_core.py", line 207, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "/path/to//python3.7/site-packages/ginza/command_line.py", line 81, in run
    print_result(line, nlp, True, output_format, output)
  File "/path/to//python3.7/site-packages/ginza/command_line.py", line 94, in print_result
    doc = nlp(line)
  File "/path/to//python3.7/site-packages/spacy/language.py", line 438, in __call__
    doc = proc(doc, **component_cfg.get(name, {}))
  File "/path/to//python3.7/site-packages/ginza/japanese_corrector.py", line 20, in __call__
    correct_dep(doc)
  File "/path/to//python3.7/site-packages/ginza/japanese_corrector.py", line 61, in correct_dep
    reading = ex_attr(token).reading + ex_attr(doc[i+1]).reading
  File "doc.pyx", line 299, in spacy.tokens.doc.Doc.__getitem__
  File "token.pxd", line 21, in spacy.tokens.token.Token.cinit
IndexError: [E040] Attempt to access token at 2, max length 2.
```

</details>

Other similar words can be processed, like:

```
echo ええ〜〜〜 | ginza  # ok
echo ええぞ〜〜 | ginza  # ok
echo ええぞ〜〜〜〜 | ginza  # ok
echo ええぞ〜〜〜 | ginza  # ng
```

How to avoid this?
Thanks a lot.

**Comment by hiroshi-matsuda-rit** [comment]
@Hiroshiba Thank you for reporting the bug! I've reproduced this error. I'd like to release the bug fix in few days.

**Comment by Hiroshiba** [comment]
Thank you for your quick response!

**Comment by hiroshi-matsuda-rit** [comment]
I fixed this issue in #67. Thank you for reporting the problems.

**Comment by Hiroshiba** [comment]
Thanks a lot!!
